samples/sensor/2_apds9960/README.rst

```diff
@@ -0,0 +1,36 @@
+.. _2_apds9960:
```

Comment:
please pick a prober  name for sample, do not use numbers.. I would just remove the existing sample we have right now or make it use the new sensor.